doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,165 @@
+successful proofs that appear after it in the script.
 
-The "gears" button submits proof terms to the Coq kernel for type checking.
-When Coq uses asynchronous processing (see Chapter :ref:`asynchronousandparallelproofprocessing`),
-proofs may have been completed without kernel-checking of generated proof terms.
-The presence of unchecked proof terms is indicated by ``Qed`` statements that
-have a subdued *being-processed* color (light blue by default), rather than the
-processed color, though their preceding proofs have the processed color.
+.. image:: ../_static/async-mode.png
+   :alt: Async mode
 
-Notice that for all these buttons, except for the "gears" button, their operations
-are also available in the menu, where their keyboard shortcuts are given.
+In the screenshot, the proof of the failed theorem can be edited (between `Proof.`
+and `Qed.`) without invalidating the theorems that follow it.  Then the modified
+proof can be reprocessed using the usual navigation operations.  The light blue
+highlight in the script indicates commands that haven't been processed.
+
+Async mode defers the final type checking step of proofs, leaving the `Qed.` marked in
+a slightly different shade of light blue to indicate this.  To complete the final
+checking, click on the "gears" button on the toolbar ("Fully check the document").
 
 Commands and templates
 ----------------------
 
 The Templates menu allows using shortcuts to insert
-commands. This is a nice way to proceed if you are not sure of the
+commands. This is a nice way to proceed if you're not sure of the
 syntax of the command you want.
 
 Moreover, from this menu you can automatically insert templates of complex
 commands like ``Fixpoint`` that you can conveniently fill afterwards.
 
 Queries
-------------
+-------
 
 .. image:: ../_static/coqide-queries.png
    :alt: CoqIDE queries
 
-We call *query* any command that does not change the current state,
-such as ``Check``, ``Search``, etc. To run such commands interactively, without
-writing them in scripts, CoqIDE offers a *query pane*. The query pane can be
-displayed on demand by using the ``View`` menu, or using the shortcut ``F1``.
-Queries can also be performed by selecting a particular phrase, then choosing an
-item from the ``Queries`` menu. The response then appears in the message window.
-The image above shows the result after selecting of the phrase
-``Nat.mul`` in the script window, and choosing ``Print`` from the ``Queries``
-menu.
+A *query* is any command that does not change the current state, such as
+:cmd:`Check`, :cmd:`Search`, etc.  The *query pane* lets you run such commands
+interactively without modifying your script. The query pane is accesible from
+the *View* menu, or using the shortcut ``F1``.
+You can also do queries by selecting a particular phrase, then choosing an
+item from the *Queries* menu. The response will appear in the message panel.
```

Comment:
```suggestion
You can also do queries by selecting a particular phrase, then choosing an
item from the *Queries* menu. The response will appear in the message panel.
```

Selecting a phrase or highlighting text with the cursor? Phrase doesn't have any technical meaning IIRC.